File: SW Expert Academy/D2/01948. 날짜 계산기/날짜 계산기.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
	int T; cin>>T;
	for(int tc=0; tc<T; tc++){
		vector<int> days(4);
		for(int i=0; i<4; cin>>days[i++]);
		
		printf("#%d ", tc+1);
		
		vector<int> plus = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30};
		int sum = 0;
		for(int i=days[0]; i<days[2]; sum += plus[i++-1]);
		sum += (days[3]-days[1]+1);
		
		printf("%d\n", sum);
	}
}

File: SW Expert Academy/D2/01948. 날짜 계산기/날짜 계산기.py
for tc in range(int(input())):
    days = list(map(int, input().split()))

    print("#%d"%(tc+1), end=' ')

    plus = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30]
    sum = 0
    for i in range(days[0], days[2]):
        sum += plus[i-1]
    sum += (days[3]-days[1]+1)

    print(sum)
